cmd/manager/main.go

```diff
@@ -64,7 +67,22 @@ func init() {
 	utilruntime.Must(asv1beta1.AddToScheme(scheme))
 }
 
+// EnvironmentEnableHypershiftImportFG is an environment key for E2E test to enable
+// the HYPERSHIFT-IMPORT feature gate.
+const EnvironmentEnableHypershiftImportFG = "ENABLE_HYPERSHIFT_IMPORT"
+
 func main() {
+	pflag.CommandLine.SetNormalizeFunc(utilflag.WordSepNormalizeFunc)
+	features.DefaultMutableFeatureGate.AddFlag(pflag.CommandLine)
+	// E2E tests use env to tweak the feature-gates flag
+	if os.Getenv(EnvironmentEnableHypershiftImportFG) == "true" {
```

Comment:
I think we should set this flag on the e2e shell script, is there any special for this?

Reply:
I tried, but it does not work. Do you know how to set it correctly?